qiskit/quantum_info/operators/symplectic/pauli_list.py

```diff
@@ -1062,3 +1064,35 @@ def from_symplectic(cls, z, x, phase=0):
         """
         base_z, base_x, base_phase = cls._from_array(z, x, phase)
         return cls(BasePauli(base_z, base_x, base_phase))
+
+    def _qw_anticommutation_graph(self):
+        """Create qubit wise anti commutation graph with edges (i, j) if i and j are not commutable.
+
+        Returns:
+            List[Tuple(int,int)]: A list of pairs of indices of the PauliList that are not commutable.
+        """
+        anti_commutation_graph = []
+        for i in range(self._num_paulis):
+            for j in range(i, self._num_paulis):
```

Comment:
If you instead do `for j in range(i + 1, ....`, then you can omit the check `i != j` below.